tests/e2e/scenarios/test_reborn_webui_v2_smoke.py

```
@@ -201,6 +201,162 @@ async def test_reborn_v2_automation_rename_persists_from_ui(
+        await expect(open_run).to_be_enabled()
+        await expect(logs).to_be_enabled()
+
+        await open_run.click()
+        await page.wait_for_url(f"**/v2/chat/{thread_id}", timeout=10000)
+
+        await page.goto(f"{reborn_v2_server}/v2/automations?token={REBORN_V2_AUTH_TOKEN}")
+        await select_automation()
+        await page.locator(SEL_V2["automation_run_logs"]).first.click()
+        await asyncio.wait_for(logs_requested.wait(), timeout=10)
+
+        assert f"/v2/logs" in page.url
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)